.ddev/web-build/Dockerfile.magerun

```diff
@@ -0,0 +1,5 @@
+RUN \
+    curl -sS -o n98-magerun.phar https://files.magerun.net/n98-magerun.phar &&  \
```

Comment:
This verifies its sha values from the internet? So if the server is compromised it can lie about the SHA it outputs aswell.
I would argue we update the n98 only when needed - with a hardcoded SHA value.

Reply:
yes, there were enough cases recently to justify being more careful with updates.